### ISSUE TITLE
Image Cropper is surprisingly slow

### ISSUE THREAD

**Issue by imagejan** [knime-ip/knip#458]
To know what's the fastest way to get a _xy_-downsampled, _channel_-cropped, _z_-projected version of an image, I assembled a small test workflow:

[Image Cropper Resizer Projector Speed Test.zip](https://github.com/knime-ip/knip/files/827617/Image.Cropper.Resizer.Projector.Speed.Test.zip)

I was a bit surprised to see that the Image Cropper performs much worse than the Image Resizer on the same input data (see the timer output; nodes are arranged in groups of three):

![image](https://cloud.githubusercontent.com/assets/2033938/23704388/ee5c5e10-0404-11e7-9953-ddc04faee92d.png)

I would have expected that simply extracting a subset of the pixel information (crop) would be faster than actually performing a linear interpolation (resize).

----

In conclusion, the fastest node arrangement to achieve the task is: _Image Resizer_ -> _Projector_ -> _Image Cropper_

**Comment by imagejan** [comment]
I should mention that for extracting a single channel, it is probably better to use a _Splitter_ instead of an _Image Cropper_ even if you only require one of the channel.

The _Splitter_ works in almost no time 😄 .

**Comment by dietzc** [comment]
here we go: https://github.com/knime-ip/knip/commit/954228212b69a3a73b4b24e74f58740f1645a0b6
I added some special cases.
